.gitignore

```diff
@@ -16,3 +16,4 @@ tags
 compiler/*.js
 compiler/**/*.js
 src/.metadata.generated.ts
+*.swp
```

Comment:
I'm somewhat reluctant to add this as it is editor specific. Out of curiosity, why isn't it in your global gitignore?

Reply:
You are probably right that this belongs to a global gitignore. That's just because I recently made switches back and forth from different editors and didn't bother changing it, simply added it where I needed.